src/utils/use-title.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+const setTitle = (title: string) => {
+  document.title = title
+}
+
+export const useTitle = (customString?: string) => {
+  const urlParams = useParams()
+  useEffect(() => {
+    const currentTitle = document.title
+    let pageTitle = `Lora`
+    if (customString) {
+      pageTitle += ` ${customString}`
+    }
+    if (urlParams.transactionId) {
+      pageTitle += ` - TxnId:${urlParams.transactionId}`
+    }
+    if (urlParams.transactionId && urlParams['*']) {
+      pageTitle += `, Inner:${urlParams['*']}`
+    }
+    if (urlParams.round) {
+      pageTitle += ` - Block:${urlParams.round}`
+    }
+    if (urlParams?.groupId) {
+      pageTitle += ` - Group:${urlParams.groupId}`
+    }
+    if (urlParams?.address) {
+      pageTitle += ` - Addr:${urlParams.address}`
```

Comment:
The verbiage we use in these pages is `Account` (e.g. check <H1> there), so the document title should probably be `Account` or `Acct` instead of `Addr` to match that

Reply:
And if we omit the `-` separators, browser tabs would show more of the title before truncating:

`Lora Acct ACSASDSADASD Mainnet` vs `Lora - Acct:ACSASDSADASD - Mainnet`

Reply:
👍 Will change to use Acct